src/mongo/delegates/Views.class.php

```diff
@@ -202,8 +217,6 @@ public function getViewForResource($resource,$viewType,$context=null)
      */
     public function getViewForResources(Array $resources,$viewType,$context=null)
```

Comment:
No, @rsinger, you're quite right: there is nothing here to do that (in this PR).

@robotrobot and I had a conversation about the starting point, and he was concerned that if we didn't consider a scheduled job early, we might pick a poor representation for spec revisions.  I've started with a scheduled job, and will move to those later, perhaps in a follow-up PR.